src/test/java/BaseTest.java

```diff
@@ -0,0 +1,48 @@
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.example.utility.AllureLogger;
+import org.example.utility.FrameworkUtility;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+
+public abstract class BaseTest extends FrameworkUtility {
```

Comment:
Yes, it's good idea to move base test from src/.../utility package to test/java package. Well done!